best-practices/client-side/framework/react/components/media-cropper/src/components/media-cropper/renderers/video.css

```diff
@@ -0,0 +1,32 @@
+.video-renderer {
```

Comment:
Would there be any value in making these styles configurable with props?

Reply:
I think that the colors for sure being configurable makes sense.